test_commands.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/sh
+# Test script for testing command entry points
+
+name="oval-graph"
+test_file_src="./tests/test_data/ssg-fedora-ds-arf.xml"
+tmp_dir_src="./tmp_data"
+tmp_json_file_src="${tmp_dir_src}/data.json"
+
+test_result=0
+overall_test_result=0
+
+usage() {
+    cat <<EOF
+usage: bash ./test_commands.sh
+-c    | --clean                  Removes created files
+-s    | --silent                 Disable every output of script
+-v    | --verbose                Displays details about the tests
+-io   | --install-oval-graph     Install oval-graph from source
+-h    | --help                   Brings up this menu
+EOF
+}
+
+clean=false
+verbose=false
+silent=false
+oval_graph_install=false
+
+for arg in "$@"; do
+    case $arg in
+    -c | --clean)
+        shift
+        clean=true
+        ;;
+    -s | --silent)
+        shift
+        silent=true
+        ;;
+    -v | --verbose)
+        shift
+        verbose=true
+        ;;
+    -io | --install-oval-graph)
+        shift
+        oval_graph_install=true
+        ;;
+    -h | --help)
+        usage
+        exit
+        ;;
+    *)
+        echo "Unknown argument ->${arg}<-"
+        exit 1
+        ;;
+    esac
+done
+
+if [ $silent = true ] && [ $verbose = true ]; then
+    echo "Excluding parameters --silent and --verbose"
+    usage
+    exit 1
+fi
+
+test_if_is_instaled_oval_graph() {
+    rpm -q $name >/dev/null 2>&1
+    is_rpm_installed=$?
+    pip list | grep -F $name
+    is_pip_package_installed=$?
+    if [ ! $is_rpm_installed ] || [ ! $is_pip_package_installed ]; then
+        echo "$name NOT installed"
+        exit 1
+    fi
+}
+
+install_package_from_source() {
+    install_pip="pip install --upgrade pip"
+    install_oval_graph="pip install -e ."
+
+    if [ $silent = false ] && [ $verbose = false ]; then
+        echo "Start install"
+        $install_pip >/dev/null
+        $install_oval_graph >/dev/null
+        echo "End install"
+    elif [ $verbose = true ] && [ $silent = false ]; then
+        echo "Start install"
+        install_pip
+        install_oval_graph
+        echo "End install"
+    else
+        install_pip >/dev/null 2>&1
```

Comment:
Earlier, you use the `$install_pip >/dev/null` redirection, and here you use `install_pip >/dev/null 2>&1`, which is inconsistent. The `&>` redirection (that works only in Bash) redirects both stdout and stderr.